backend/testfiles/execution/stdlib/httpclient.dark

```diff
@@ -18,17 +18,44 @@ module NoInternal =
   PACKAGE.Darklang.Stdlib.HttpClient.bearerToken "YWxhZGRpbjpvcGVuc2VzYW1l" = (("authorization",
                                                                                 "bearer YWxhZGRpbjpvcGVuc2VzYW1l"))
 
-  Builtin.HttpClient.basicAuth "username" "password" = ("authorization",
-                                                        "basic dXNlcm5hbWU6cGFzc3dvcmQ=")
-
-  Builtin.HttpClient.basicAuth "" "" = ("authorization", "basic Og==")
-
-  Builtin.HttpClient.basicAuth "-" "" = Builtin.Test.derrorMessage
+  PACKAGE.Darklang.Stdlib.HttpClient.basicAuth "username" "password" = PACKAGE
+    .Darklang
+    .Stdlib
+    .Result
+    .Result
+    .Ok(("authorization", "basic dXNlcm5hbWU6cGFzc3dvcmQ"))
+
+  PACKAGE.Darklang.Stdlib.HttpClient.basicAuth "" "" = PACKAGE
+    .Darklang
+    .Stdlib
+    .Result
+    .Result
+    .Ok(("authorization", "basic Og"))
+
+  PACKAGE.Darklang.Stdlib.HttpClient.basicAuth "-" "" = PACKAGE.Darklang.Stdlib.Result.Result.Error
     "Username cannot contain a hyphen"
 
-  Builtin.HttpClient.basicAuth "" "-" = ("authorization", "basic Oi0=")
-  Builtin.HttpClient.basicAuth ":" "" = ("authorization", "basic Ojo=")
-  Builtin.HttpClient.basicAuth "" ":" = ("authorization", "basic Ojo=")
```

Comment:
the change from Ojo= to Ojo is because of our String.base64Encode it replaces = with ""